src/System.IO.FileSystem/tests/File/Create.cs

```diff
@@ -235,4 +235,41 @@ public void UnixWhitespacePath()
 
         #endregion
     }
+
+    public class File_Create_str_i : File_Create_str
+    {
+        protected const int DefaultBufferSize = 4096;
+
+        public override FileStream Create(string path)
+        {
+            return File.Create(path, DefaultBufferSize);
+        }
+
+        public virtual FileStream Create(string path, int bufferSize)
+        {
+            return File.Create(path, bufferSize);
+        }
+
+        [Fact]
+        public void NegativeBuffer()
+        {
+            Assert.Throws<ArgumentOutOfRangeException>(() => Create(GetTestFilePath(), -1));
+            Assert.Throws<ArgumentOutOfRangeException>(() => Create(GetTestFilePath(), -100));
+        }
+    }
+
+    public class File_Create_str_i_fo : File_Create_str_i
+    {
+        protected const bool DefaultUseAsync = true;
+
+        public override FileStream Create(string path)
+        {
+            return File.Create(path, DefaultBufferSize, DefaultUseAsync ? FileOptions.Asynchronous : FileOptions.None);
```

Comment:
yeah, conditional was a bad choice. If I'm gonna hardcode DefaultUseAsync, I might as well make it of type "FileOptions", not bool. 

It would probably be more concise to just put the default value in the function call itself though.